openfisca_core/simulations.py

```diff
@@ -132,12 +132,13 @@ def clone(self, debug = False, debug_all = False, trace = False):
             if key not in ('debug', 'debug_all', 'trace', 'entities'):
```

Comment:
> Add `persons` ?

We redefine `new.person` just further, so it's useless. I remove `entities` as well, for the same reason.